requirements/constraints.txt

```diff
@@ -118,3 +118,6 @@ python3-saml<1.10.0
 # With greater version these tests `common.djangoapps.terrain.stubs.tests` becomes flaky on jenkins.
 httpretty<1.0
 
+# pygments was constrained by diff-cover see https://github.com/Bachmann1234/diff_cover/commit/01f91760321cee1ad28cfa0d801c4acd8b9765a6
+pygments==2.6.1
```

Comment:
Can we instead constrain diff-cover to avoid the overly precise pinning that the newest diff-cover adds?

It would also be good to include a Jira ticket here (for unpinning) and/or a Github issue (on diff-cover) to track, if possible.